### DISCUSSION TITLE
ElasticSearch error in Browser

### DISCUSSION THREAD

**Discussion by ashokdev7** [Security-Onion-Solutions/securityonion#6918] [Unsupported Versions] [answered]
Hi I can access Elastic search indices using terminal
curl -sk https://ip:9200/_cat/indices -u username:password

But How can i open this in browser
I tried https://ip:9200/_cat/indices , also https://username:password@ip:9200/_cat/indices
Still not being able to access I get ERR_SSL_KEY_USAGE_INCOMATIBLE
I even tried changing filebeat.yaml , ssl.enabled="false"
How to solve this or  how can I remove https and use http

**Comment by dougburks** [comment, marked as answer]
If you want to search Elastic in your browser, perhaps you are looking for our Hunt interface or Kibana:
https://docs.securityonion.net/en/2.3/hunt.html
https://docs.securityonion.net/en/2.3/kibana.html
